core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1511,15 +1511,6 @@ public String toString() {
           .setDefaultValue(19200)
           .setScope(Scope.ALL)
           .build();
-  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_STORAGE_LEVEL =
```

Comment:
Is there a way to warn user of the deprecation of this property?

Reply:
Oh yeah, i wll leave the property key there, i remember it's bunch processed during release

Reply:
Will add a @deprecated to the key